### PR TITLE
👩‍💻 Add markdown formatting

### LINKED ISSUE #69: 👩‍💻 Code Formatting
The following output from the agent looks difficult to read the code

Screenshot 2023-04-12 225054

### PR DESCRIPTION
**Pull request by asim-shrestha**
closes #69

### CODE REVIEW

File: src/styles/globals.css [hunk 1]
```diff
@@ -25,9 +26,31 @@
   right: 0;
 }
 
+/* Used for code / table formatting within messages */
+pre {
```
Comment (awtkns) [inline src/styles/globals.css:30]: Any way to make these more specific in the future? We should probably try and avoid globals going forward

**Review by awtkns — APPROVED** [review]

**Comment by awtkns** [comment]
Looks good 
image

File: src/styles/globals.css (hunk 1 above)
Comment (asim-shrestha) [reply src/styles/globals.css:30]: true 🚀

Comment (asim-shrestha) [reply src/styles/globals.css:30]: Not sure, haven't looked deeply but can explore when the time comes

**Comment by awtkns** [comment]
@asim-shrestha i did notice tasks now start on a new line, is that intentional?

**Pull request merged**